Bicep file that creates a basic WVD Backplane

fberson opened the pull request:
This Bicep creates a WVD Backplane in Azure consisting of :
- WVD Hostpool
- WVD AppGroup
- WVD Workspace
- A configured relationship between these objects.

anthony-c-martin commented:
@fberson, thank you for contributing!

Just FYI to fix up the test failures:
1. The build expects to see a .json file for each .bicep file checked in, so you'll need to include those, even for the individual module files. If you have .NET 5 installed locally, you can fix all at once with `dotnet test -- 'TestRunParameters.Parameter(name=\"SetBaseLine\", value=\"true\")'` in the root of the repo, otherwise you can just run `bicep build` for each of the module files.
1. There is a 'missing type' warning which is also blocking check-in. You can fix that by copying one of the entries here and adding one for `Microsoft.DesktopVirtualization/hostpools/providers/diagnosticSettings@2017-05-01-preview`:
https://github.com/Azure/bicep/blob/main/src/Bicep.Core.Samples/ExamplesTests.cs#L84-L101

fberson commented:
@anthony-c-martin I added the required .json file for the .bicep file and reverted the module project to create individual later, but I still see failures? Can you help?

anthony-c-martin commented:
> @anthony-c-martin I added the required .json file for the .bicep file and reverted the module project to create individual later, but I still see failures? Can you help?

Ah yes, I forgot to mention. The tests are also very fussy about the format of the `.bicep` file - just to enforce consistency in the samples. I used the VSCode "Format Document" command on your file and have checked in the updated file to your branch.

We've got some info on the various Examples validations here: https://github.com/Azure/bicep/blob/main/CONTRIBUTING.md#example-files

anthony-c-martin reviewed (approved):
Looks great! Thank you for contributing!

fberson commented:
Thanks @anthony-c-martin !
